cosmos/operators/docker.py

```diff
@@ -52,6 +52,10 @@ def build_command(self, context: Context, cmd_flags: list[str] | None = None) ->
         self.environment = {**env_vars, **self.environment}  # type: ignore[has-type]
         self.command = dbt_cmd
 
+    def execute(self, context: Context) -> None:
+        result = self.build_and_run_cmd(context=context)
+        logger.info(result.output)
```

Comment:
wdyt about adding the `logger.info` in `build_and_run_cmd()`? it seems we log the output every time the function is called, so this could help with code duplication

Reply:
Good idea, I'll do this, @jlaneve !